samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget-e2etests/Helpers/IOktaSdkHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Okta.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace embedded_sign_in_widget_e2etests.Helpers
+{
+    public interface IOktaSdkHelper
+    {
+        Task<IUser> CreateActiveUser(string userName, string firstName, string password);
+        Task AddUserToGroup(IUser user, string groupName);
+        Task DeleteUserAsync(string email);
+        Task<IUserFactor> EnrollPhoneFactor(string emailAddress, string phoneNumber);
```

Comment:
I believe we don't need factor-related methods

Reply:
Agree, we only need `CreateActiveUser` function for this test.

Reply:
Correction: we also need `DeleteUserAsync`

Reply:
👍